boilerplate/flyte/end2end/Makefile

```diff
@@ -0,0 +1,9 @@
+# WARNING: THIS FILE IS MANAGED IN THE 'BOILERPLATE' REPO AND COPIED TO OTHER REPOSITORIES.
+# ONLY EDIT THIS FILE FROM WITHIN THE 'FLYTEORG/BOILERPLATE' REPOSITORY:
+#
+# TO OPT OUT OF UPDATES, SEE https://github.com/flyteorg/boilerplate/blob/master/Readme.rst
+
+.PHONY: end2end_execute
+end2end_execute:
+	chmod a+x ./boilerplate/flyte/end2end/end2end.sh
```

Comment:
Is this still needed?  I thought not?

Reply:
we can leave it for now and boilerplate automation will automatically create a PR. Fix is needed in boilerplate